ocs_ci/framework/conf/default_config.yaml

```diff
@@ -19,14 +19,14 @@ RUN:
   run_id: null  # this will be redefined in the execution
   kubeconfig_location: 'auth/kubeconfig' # relative from cluster_dir
   cli_params: {}  # this will be filled with CLI parameters data
-  client_version: '4.1.4'
+  client_version: '4.2.0-0.nightly-2019-07-30-073644'
```

Comment:
Any reason to have this hardcoded like this?

Also what is the reason for the default to be a nightly build and not CI build? What would we suggest the team to prefer?

Reply:
> Any reason to have this hardcoded like this?

No reason other than this is the current way we are handling OCP client and installer versions. Note this is the default config that can be overwritten by passing a different value through with `--ocsci-conf` so they aren't "hardcoded" as much as they are default values.

> Also what is the reason for the default to be a nightly build and not CI build? What would we suggest the team to prefer?

This was the latest version I locally tested with, so I stuck with it. The point is to ensure we are testing 4.2. I have no _personal_ preference between CI and Nightly.

Reply:
Ok, so this looks as a safe default and we expect people to check which version to use?

Reply:
> we expect people to check which version to use?

I'm not sure what you mean by this. If users need to test using a specific version of OCP client or installer then yeah they should be checking what version we are using by default and override if necessary.

Reply:
I see what Martin is saying,  Is there a -latest-nightly tag that one can use?

Reply:
@vasukulkarni we can get latest 4.2 nightly build from https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4.2.0-0.nightly/latest

Reply:
from above link we can get download link of latest binary and we can download and deploy cluster i just made a small script to get downloadlink and download oc-client and oc-installer 
https://gist.github.com/prsurve/dda470d8ad42e17af8f1a026ce9d63a7

Reply:
If we want to default to using the latest nightly then let's create an issue for that and I can either update this PR to handle it or do it in a subsequent PR. Defaulting to the latest CI or nightly release wasn't a requirement so much as simply moving to 4.2 was so I didn't go that route.

Reply:
IMO there's no reason for consuming latest.
We are not doing QA for OCP and should not waste time on testing the failures landed there. 
Let's stick with a stable version and once in a while, move on to a new one